templates/digital-asset-template/move/sources/launchpad.move

```diff
@@ -262,12 +264,30 @@ module launchpad_addr::launchpad {
         let stage = &mint_stage::execute_earliest_stage(sender, collection_obj, 1);
         assert!(option::is_some(stage), E_NO_ACTIVE_STAGES);
 
-        let mint_fee = get_mint_fee(collection_obj, *option::borrow(stage));
+        let mint_fee = get_mint_fee_per_nft(collection_obj, *option::borrow(stage));
         pay_for_mint(sender, mint_fee);
 
         mint_nft_internal(sender_addr, collection_obj, mint_fee);
     }
 
+    public entry fun batch_mint_nft(
+        sender: &signer,
+        collection_obj: Object<Collection>,
+        amount: u64,
+    ) acquires CollectionConfig, CollectionOwnerObjConfig, Config {
+        let sender_addr = signer::address_of(sender);
+
+        let stage = &mint_stage::execute_earliest_stage(sender, collection_obj, amount);
```

Comment:
what does "stage" mean?

Reply:
mint stage

Reply:
thoughts on calling it `mint_stage`? 😄

Reply:
sadly that function is defined in `mint_stage` module in `token-minter`, not in our template 😢